lib/instances/BaseInstance.lua

```diff
@@ -204,6 +212,16 @@ function BaseInstance.prototype:WaitForChild(name, delay)
 	return self:FindFirstChild(name)
 end
 
+function BaseInstance.prototype:_PropagateAncestryChanged(child, parent)
```

Comment:
This feels off, but I don't know if there's a better place to put internal methods, so it's probably fine.